model_analyzer/config/generate/quick_run_config_generator.py

```diff
@@ -511,9 +511,12 @@ def _get_next_perf_analyzer_config(
 
         perf_analyzer_config.update_config_from_profile_config(model_name, self._config)
 
-        concurrency = self._calculate_concurrency(dimension_values)
+        perf_config_params = {"batch-size": 1}
```

Comment:
I feel like it'd be cleaner if the PerfAnalyzerConfig() constructor initialized `batch-size: 1`. Seems like we are always needed to add this in the code.

Reply:
This is done